consul/helper.go

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package consul
```

Comment:
Do you see a need to have a separate set of test cases for this file? Especially the `FormatKeys` method

Reply:
no. will add that.